### ISSUE TITLE
Support working directory?

### ISSUE THREAD

**Issue by NullVoxPopuli** [mainmatter/ember-asset-size-action#6]
My ember app isn't in the root of my repo:

https://github.com/NullVoxPopuli/emberclear/pull/279/checks?check_run_id=386020382

This is the config I used:
```yaml
name: Frontend Asset Sizes

on: [pull_request]

jobs:
  build:

    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v2-beta
      with:
        fetch-depth: 0
    - uses: simplabs/ember-asset-size-action@v1
      working-directory: packages/frontend
      with:
        repo-token: "${{ secrets.GITHUB_TOKEN }}"
```
working directory is supported by many other actions, afaik. 

Maybe that's just me not knowing how GH Actions syntax works. idk

I then tried this:
```yaml
name: Frontend Asset Sizes

on: [pull_request]

env:
  cwd: packages/frontend

jobs:
  build:

    runs-on: ubuntu-latest

    steps:
    - uses: actions/checkout@v2-beta
      with:
        fetch-depth: 0

    - uses: rwjblue/setup-volta@v1
      with:
        working-directory: ${{ env.cwd }}

    - uses: simplabs/ember-asset-size-action@v1
      with:
        repo-token: "${{ secrets.GITHUB_TOKEN }}"
        working-directory: ${{ env.cwd }}
```

which then gave the error: 
![image](https://user-images.githubusercontent.com/199018/72227224-f108a280-3567-11ea-8992-34aa8ccc4d00.png)
This looks like it's ignoring the working directory setting?
🤷‍♂

**Comment by mansona** [comment]
so nothing in actions is "standard" so the `working-directory` implementation you're talking about must be implemented by these other actions you're talking about. Do you have an example? 

I guess it makes sense to add it 🤔 let me know if you want to have a crack at it and we can discuss a PR 👍

**Comment by NullVoxPopuli** [comment]
hmm, if working-directory must be implemented by everything, would it be better to just cd to everything first?
I guess that still doesn't work for actions that "do the thing" for you. _hmm_.

Yeah, next I look at this, I'll see if I can poke around the source and compare it to something else that I know supports working-directory.

**Comment by lifeart** [comment]
Faces same issue

**Comment by NullVoxPopuli** [comment]
I have a PR here: but there is to way to know if it works

https://github.com/simplabs/ember-asset-size-action/pull/8

:(

See this thread: https://github.community/t5/GitHub-Actions/401-with-octokit-using-the-actions-provided-taken-Bad/m-p/53785#M8964

**Comment by mansona** [comment]
Hey sorry I missed your PR @NullVoxPopuli, I'll take a look on Friday 👍 sorry for the delay

**Comment by alexlafroscia** [comment]
I would love to see something like this supported! I have a mono-repo where I would like to use this action for each app/addon in the mono-repo, and I think that just having support for pointing to an app would be enough to handle

**Comment by btecu** [comment]
This would also support multiple applications in the same repository.

**Comment by mansona** [comment]
This is now fixed since #67 was merged
